single-rai-job/src/run_rai.py

```diff
@@ -133,6 +135,9 @@ class PropertyKeyValues:
     # Property format to indicate presence of a tool
     RAI_INSIGHTS_TOOL_KEY_FORMAT = "_azureml.responsibleai.rai_insights.has_{0}"
 
+    # Property to track the data type
+    RAI_INSIGHTS_DATA_TYPE_KEY = "_azureml.responsibleai.rai_insights.data_type"
```

Comment:
where is this used

Reply:
in the TerminalContainer in workspace-portal, it needs to know if it's a forecasting task type to decide whether or not to use a separate docker container for serving the model